osu.Game/Overlays/Profile/Header/PreviousUsernamesContainer.cs

```diff
@@ -0,0 +1,181 @@
+                return;
+            }
+
+            Hide();
+        }
+
+        protected override void OnHoverLost(HoverLostEvent e)
+        {
+            base.OnHoverLost(e);
+            contentContainer.Hide();
+            this.MoveToY(0, duration, Easing.OutQuint);
+        }
+
+        private class ContentContainer : VisibilityContainer
+        {
+            private const int header_height = 25;
+            private const int content_padding = 60;
+
+            public readonly TextFlowContainer Usernames;
+
+            private readonly Box background;
+
+            public ContentContainer()
+            {
+                AutoSizeAxes = Axes.Y;
+                RelativeSizeAxes = Axes.X;
+                Masking = true;
+                AlwaysPresent = true;
+                CornerRadius = 5;
+                Children = new Drawable[]
+                {
+                    background = new Box
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                    },
+                    new FillFlowContainer
+                    {
+                        RelativeSizeAxes = Axes.X,
+                        AutoSizeAxes = Axes.Y,
+                        Direction = FillDirection.Vertical,
+                        Spacing = new Vector2(0, 5),
+                        Children = new Drawable[]
+                        {
+                            new Container
+                            {
+                                RelativeSizeAxes = Axes.X,
+                                Height = header_height,
+                                Children = new Drawable[]
+                                {
+                                    new SpriteText
+                                    {
+                                        Anchor = Anchor.BottomLeft,
+                                        Origin = Anchor.BottomLeft,
+                                        Padding = new MarginPadding { Left = content_padding },
+                                        Text = @"formerly known as",
+                                        Font = OsuFont.GetFont(size: 14, italics: true)
+                                    }
+                                }
+                            },
+                            Usernames = new TextFlowContainer
+                            {
+                                RelativeSizeAxes = Axes.X,
+                                AutoSizeAxes = Axes.Y,
+                                Direction = FillDirection.Full,
+                                Spacing = new Vector2(5, 5),
+                                Padding = new MarginPadding { Left = content_padding, Bottom = margin },
+                            },
+                        }
+                    }
+                };
+            }
+
+            [BackgroundDependencyLoader]
+            private void load(OsuColour colours)
+            {
+                background.Colour = colours.GreySeafoamDarker;
```

Comment:
Black?

Reply:
Well, it's not black in the current "pink" design, so I assume it's not black and here